docs/source/tutorial.rst

```diff
@@ -0,0 +1,126 @@
+Tutorial
+--------
+
+For this tutorial we are going to use a well known case file from `NIST <https://cfreds.nist.gov/all/NIST/HackingCase>`_.
+In this tutorial we assume Linux is used, on other operating systems most steps are the same but details may differ.
+First of all, we install Python and create a venv:
+
+.. code-block:: console
+
+    $ python3 -m venv dissect
+
+Now activate the virtual environment to get some work done:
+
+
+.. code-block:: console
+
+    $ source dissect/bin/activate
+    
+
+Let's download the case files. At the time of writing they are hosted at nist.gov.
+On Linux:
+
+.. code-block:: console
+
+    $ for i in {1..8}; do curl https://cfreds-archive.nist.gov/images/hacking-dd/SCHARDT.00$i -o SCHARDT.00$i; done
+    
+.. note ::
+
+    You can also download them manually of course.
+    
+Now, we are going to do some basic operations on this image. If you like, you can merge them together first,
+although this is not strictly necessary:
+
+.. code-block:: console
+
+    $ for i in `ls SCHARDT.00*`; do cat $i >> SCHARDT.img; done
+
+
+To get a brief summary of the forensic image, we use :doc:`target-info <tools/target-info>` like this:
+
+.. code-block:: console
+
+    $ target-info SCHARDT.img
+
+The result will be something like this:
+
+.. code-block:: console
+
+    Disks
+    - <Disk type="RawContainer" size="4871301120">
+
+    Volumes
+    - <Volume name="part_00007e00" size="4869333504" fs="NtfsFilesystem">
+
+    Hostname       : N-1A9ODN6ZXK4LQ
+    Domain         : None
+    Ips            : 192.168.1.111
+    Os family      : windows
+    Os version     : Microsoft Windows XP (NT 5.1) 2600
+    Architecture   : x86_32-win32
+    Language       : 
+    Timezone       : America/Chicago
+    Install date   : 2004-08-19 22:48:27+00:00
+    Last activity  : 2004-08-27 15:46:33.820240+00:00
+
+To get the list of user accounts on this machine we use :doc:`target-query <target-query>`.
```

Comment:
I would suggest to split this up. Explain you can use target-query with the users plugin. Display target-query img.img -f users and give output. Then explain you can combine it with rdump. Display target-query | rdump command with output.

This approach will make it way easier for a reader to follow.

Reply:
The frontpage already shows `-f users `. There is some synergy between the pages of the manual to let the reader flow more naturally from one subject into another. I guess this gets lost in github.